scsi_helpers.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2014 Red Hat, Inc.
+ *
+ * This file is licensed to you under your choice of the GNU Lesser
+ * General Public License, version 2.1 or any later version (LGPLv2.1 or
+ * later), or the Apache License 2.0.
+ */
+
+/*
+ * APIs for both libtcmu users and tcmu-runner plugins to use.
```

Comment:
Fix up the comment and drop libtcmu since we are only using them in tcmu-runner initially.

Also just name the files scsi.c/.h.

Reply:
OK, thanks.